openprescribing/dmd/models.py

```diff
@@ -164,6 +164,10 @@ class DMDVmpp(models.Model):
     qty_uomcd = models.BigIntegerField(blank=True, null=True)
     combpackcd = models.BigIntegerField(blank=True, null=True)
 
+    @property
+    def products(self):
+        return DMDProduct.objects.filter(vpid=self.vpid)
```

Comment:
I'm a bit wary of putting database access behind a `@property`.  Would it be possible to add an explicit `ForeignKey`?

Reply:
Actually, it doesn't seem to be used anywhere, so I'll delete it